src/main/java/com/conveyal/datatools/manager/controllers/api/UserController.java

```diff
@@ -300,6 +304,45 @@ private static Object getRecentActivity(Request req, Response res) {
         return activityList;
     }
 
+    /**
+     * Resends the user confirmation email for a given user
+     */
+    private static ObjectNode resendEmailConfirmation(Request req, Response res) {
+        // verify if the request is legit. Should only come from an application administrator or the user for which the
+        // account belongs to.
+        Auth0UserProfile userProfile = req.attribute("user");
+        if (userProfile.canAdministerApplication() || userProfile.getUser_id().equals(req.params("id"))) {
```

Comment:
When would this request be coming from an application administrator? Perhaps this needs a TODO describing that this never actually happens in the current UI, but is some kind of future proofing? It might should just be removed though. Curious to hear your thoughts.

Reply:
Yeah, it's not currently used anywhere in the current app, but it sort of seemed like something a thing that the app admin would be able to do. I'll just remove it and we can add it back later if needed.